docs/src/lib/components/demos/slider-demo.svelte

```diff
@@ -11,18 +11,16 @@
 		bind:value
 		class="relative flex w-full touch-none select-none items-center"
 	>
-		{#snippet children()}
-			<span
-				class="bg-dark-10 relative h-2 w-full grow cursor-pointer overflow-hidden rounded-full"
-			>
-				<Slider.Range class="bg-foreground absolute h-full" />
-			</span>
-			<Slider.Thumb
-				index={0}
-				class={cn(
-					"border-border-input bg-background hover:border-dark-40 focus-visible:ring-foreground dark:bg-foreground dark:shadow-card focus-visible:outline-hidden data-active:scale-[0.98] data-active:border-dark-40 block size-[25px] cursor-pointer rounded-full border shadow-sm transition-colors focus-visible:ring-2 focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50"
-				)}
-			/>
-		{/snippet}
```

Comment:
making a note here incase this `children` snippet was added here intentionally for the purposes of the demo while not necessarily being needed